x/twasm/contract/incoming_msgs.go

```diff
@@ -100,7 +100,6 @@ func (p ExecuteGovProposal) GetProposalContent(sender sdk.AccAddress) govtypes.C
 	case p.Proposal.MigrateContract != nil:
 		p.Proposal.MigrateContract.Title = p.Title
 		p.Proposal.MigrateContract.Description = p.Description
-		p.Proposal.MigrateContract.RunAs = sender.String()
```

Comment:
Huh, I removed that field in another PR, surprised this didn't show up (I guess it would have broken).
Anyway, good to remove